iree/modules/check/native_module.cc

```diff
@@ -83,6 +84,20 @@ bool AlmostEqByteSpan(iree_byte_span_t lhs_bytes, iree_byte_span_t rhs_bytes) {
   return true;
 }
 
+bool AlmostEqByteSpanF16(iree_byte_span_t lhs_bytes,
+                         iree_byte_span_t rhs_bytes) {
+  auto lhs_span = AbslSpan<uint16_t>(lhs_bytes);
+  auto rhs_span = AbslSpan<uint16_t>(rhs_bytes);
+  assert(lhs_span.size() == rhs_span.size());
+  for (int i = 0; i < lhs_span.size(); ++i) {
+    if (fabs(half_float::detail::half2float<float>(lhs_span[i]) -
```

Comment:
out of curiosity: `detail` usually indicates a private implementation detail - is there a public API that does the same, or is this the only way?

Reply:
Ah good catch Ben, yeah it would be nice not to use a detail namespace here :-)

Reply:
Yes, that's a good point but I don't think this library expose any other way to convert half to float. I think the idea is that you should be using their internal type for half type. However I only want conversion in my case. I see it used like that in a few places in google internal code.

Reply:
makes sense!